code/datums/weather/weather.dm

```diff
@@ -171,6 +174,7 @@
 /datum/weather/proc/end()
 	if(stage == END_STAGE)
 		return 1
```

Comment:
```suggestion
		return TRUE
```

Reply:
There's nothing that actually uses this return var, I'm just gonna nuke it